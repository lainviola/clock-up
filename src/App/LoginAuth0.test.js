import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import LoginAuth0 from './LoginAuth0'

describe('Component: LoginAuth0', () => {
  it('snapshot', () => {
    const wrapper = shallow(<LoginAuth0 />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
