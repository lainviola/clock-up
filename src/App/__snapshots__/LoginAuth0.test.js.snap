// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: LoginAuth0 snapshot 1`] = `
<Route
  render={[Function]}
/>
`;
